argus/introspection/cloud/windows.py

```diff
@@ -68,6 +68,19 @@ def _escape_path(path):
     return path
 
 
+def _getips_v4_v6(ips_as_string):
```

Comment:
As I can see, you are using only the first and the last entry, so why does it have to be a list? Return a namedtuple, which can be used as this:

ip.gateway6
ip.gateway
ip.address
ip.address6

Cleaner, no need to know the position of the specific IP.